src/MembershipsSDKAdapter.js

```diff
@@ -16,11 +16,12 @@ import {DestinationType, MembershipsAdapter} from '@webex/component-adapter-inte
  * @param {object} members Members object from meeting, keyed by ID
  * @returns {Array} List of active users in a meeting
  */
-function getActiveMembers(members) {
```

Comment:
Since you're removing the active members return, have you verified that this change does not break the "waiting for others..." state in the meeting component?

Reply:
good catch, thank you

Reply:
it's now fixed here https://github.com/webex/components/pull/165/commits/78e1f2699f71d1d8d087a565c6489e19718fed71#diff-eb5454f98c186e1cdd3067955c91f6ab2a64b7eee2de9026c49ee64901cf5f97R39

Reply:
it's here https://github.com/webex/components/pull/165/files#diff-eb5454f98c186e1cdd3067955c91f6ab2a64b7eee2de9026c49ee64901cf5f97R39

Reply:
Perfect!